_layouts/documentation.html

```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+bootstrap: true
+---
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-sm-3 col-lg-2">
```

Comment:
I'd rather put col-lg-3 (the result is better when displaying computation engines)